xyz.safewords.midiconn.yml

```diff
@@ -0,0 +1,63 @@
+command: midiconn
+id: xyz.safeworlds.midiconn
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+modules:
+  - name: SDL2
+    buildsystem: autotools
+    sources:
+      - type: git
+        url: https://github.com/libsdl-org/SDL
+        tag: release-2.26.1
+  - name: fmt
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DFMT_DOC=OFF
+      - -DFMT_INSTALL=ON
+      - -DFMT_TEST=OFF 
+      - -DCMAKE_BUILD_TYPE=Release
+    sources:
+      - type: git
+        url: https://github.com/fmtlib/fmt
+        tag: '8.1.1'
+  - name: spdlog
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+    sources:
+      - type: git
+        url: https://github.com/gabime/spdlog
+        tag: v1.10.0
+  - name: rtmidi
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DRTMIDI_API_JACK=OFF
+      - -DRTMIDI_BUILD_STATIC_LIBS=ON
+      - -DCMAKE_BUILD_TYPE=Release
+    sources:
+      - type: git
+        url: https://github.com/thestk/rtmidi
+        commit: 806e18f575b68c23b26f9398e1b6866b335b5308
+  - name: midiconn
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+    sources:
+      - type: git
+        url: https://gitlab.com/mfep/midiconn.git
+        tag: 0.1.0
+cleanup:
```

Comment:
cleanup and finish-args are usually about modules.

Reply:
Fixed as requested.